internal/pkg/template/templates/environment/cf.yml

```diff
@@ -158,14 +175,6 @@ Resources:
       GroupId: !Ref EnvironmentSecurityGroup
       IpProtocol: -1
       SourceSecurityGroupId: !Ref InternalLoadBalancerSecurityGroup
-  InternalALBIngressFromEnvironmentSecurityGroup:
```

Comment:
Can we still keep the old security group settings?
I'd want us to default to the principle of least privileged although it doesn't meet the customer's usecase. Instead for now, we can Output the `InternalLoadBalancerSecurityGroup` so that people can write their own addons augmenting the permission for the internal ALB.

We can follow up with folks on the github issues to make sure an addon is an okay tradeoff - and very soon they'll be able to customize it with the env manifest